File: backend/src/main/java/um/edu/ar/backend/infrastructure/web/dto/VentaResumenResponse.java
package um.edu.ar.backend.infrastructure.web.dto;

import java.time.Instant;

public record VentaResumenResponse(
        Long eventoId,
        Long ventaId,
        Instant fechaVenta,
        boolean resultado,
        String descripcion,
        double precioVenta,
        int cantidadAsientos
) {}

File: cliente_movil/composeApp/src/commonMain/kotlin/um/edu/ar/model/VentaResumenResponse.kt
package um.edu.ar.model

import kotlinx.serialization.Serializable

@Serializable
data class VentaResumenResponse(
    val eventoId: Long,
    val ventaId: Long? = null,
    val fechaVenta: String,
    val resultado: Boolean,
    val descripcion: String,
    val precioVenta: Double,
    val cantidadAsientos: Int
)
